fs/fshttp/http.go

```diff
@@ -72,16 +72,20 @@ func NewTransportCustom(ctx context.Context, customize func(*http.Transport)) ht
 		t.TLSClientConfig.Certificates = []tls.Certificate{cert}
 	}
 
-	// Load CA cert
-	if ci.CaCert != "" {
-		caCert, err := os.ReadFile(ci.CaCert)
-		if err != nil {
-			log.Fatalf("Failed to read --ca-cert: %v", err)
-		}
+	// Load CA certs
+	if ci.CaCert != nil && len(ci.CaCert) != 0 {
+
 		caCertPool := x509.NewCertPool()
-		ok := caCertPool.AppendCertsFromPEM(caCert)
-		if !ok {
-			log.Fatalf("Failed to add certificates from --ca-cert")
+
+		for i := 0; i < len(ci.CaCert); i++ {
+			caCert, err := os.ReadFile(ci.CaCert[i])
+			if err != nil {
+				log.Fatalf("Failed to read --ca-cert file %s : %v", ci.CaCert[i], err)
```

Comment:
I'd usually use `%q` instead of %s here which will put the string in quotes. Helps to see if you've left trailing spaces etc. So something like

    log.Fatalf("Failed to read --ca-cert %q: %v", cert, err)